packages/odyssey-storybook/src/components/odyssey-mui/Tabs/Tabs.stories.tsx

```diff
@@ -43,19 +51,46 @@ const storybookMeta: Meta<TabsProps & TabItemProps> = {
 
 export default storybookMeta;
 
+const selectTab = async (
+  canvasElement: HTMLElement,
+  step: StepFunction<ReactRenderer, TabItemProps>,
+  action: string,
+  tabName: string
+) => {
+  await step("select the tab", async () => {
+    await axeRun(action);
+    await sleep();
+
+    const canvas = within(canvasElement);
+    const tab = canvas.getByText(tabName);
+    await userEvent.click(tab);
+    await userEvent.tab();
+    await expect(
+      canvas.getByText(`Information about ${tabName}`)
+    ).not.toBeNull();
+
+    if (action === "Tab Disabled") {
+      const disabledTab = canvas.getByText("Disabled Tab");
+      await userEvent.click(disabledTab);
+      const tabData = screen.queryByText("Tab is disabled");
+      expect(tabData).toBeNull();
+    }
+  });
+};
+
 const DefaultTemplate: StoryObj<TabItemProps> = {
   render: function C(args) {
     const tabs: TabItemProps[] = [];
 
     tabs.push({
       label: "Planets",
       value: "planets",
-      children: "Information about Planets.",
+      children: "Information about Planets",
     });
     tabs.push({
       label: "Moons",
       value: "moons",
-      children: "Information about Moons.",
+      children: "Information about Moons",
     });
```

Comment:
Even though it's a story, I don't like mutations at all and don't wanna have them be documented in our codebase. Can we just do `tabs = [{}, {}]` instead?